src/cloudlabs/templates/setup_script.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+{% from "includes/_form_helpers.html" import render_field %}
+
+{% block content %}
+  <form method="POST" action="{{ url_for('host.customise_setup') }}">
+    {{ form.csrf_token }}
+    {{ form.hidden_tag() }}
+    {{ render_field(form.setup_script) }}
```

Comment:
The default rendering of the text area makes it pretty small (you can only see the first 2 lines). Would be good if it could be 10 lines high by default.